src/main/java/cz/student/osydorchuk89/TaskManager.java

```diff
@@ -1,9 +1,122 @@
 package cz.student.osydorchuk89;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+import cz.common.ConsoleColors;
+
 public class TaskManager {
 
-    public static void main(final String[] args) {
+    private static List<String> options = new ArrayList<>(List.of("add", "remove", "list", "exit"));
+    private static List<List<String>> tasks = new ArrayList<>();
+    private static Scanner scanner = new Scanner(System.in);
+    private static String TASKS_FILE = "tasks.csv";
+
+    public static void main(String[] args) {
+        loadTasks();
+        System.out.println("Please select an option from the list below: " + ConsoleColors.BLUE);
+        for (String option: options) {
+            System.out.println(option);
+        }
+        System.out.println(ConsoleColors.RESET);
+        while (true) {
+            final String option = scanner.nextLine();
+            switch (option) {
```

Comment:
CLEAN-CODE: promenna option se pouziva jenom v switch, proto bych ji zrusil a nacteni dal primo do switch.

```suggestion
            switch ( scanner.nextLine().toLowerCase()) {
```